theano/gpuarray/opt.py

```diff
@@ -2219,7 +2286,7 @@ def local_gpu_svd(op, context_name, inputs, outputs):
                                'conv_gemm',
                                'gpuarray', 'fast_compile', 'fast_run')
 
-
+conv_metaopt.register(abstractconv_groupopt.query(*['+' + name for name in abstractconv_groupopt._names]).opts)
```

Comment:
Won't this register the meta-optimizer as an optimizer for itself? I think I'd move the `abstractconv_groupopt.register('conv_metaopt', ...)` to after this line. See my original code: https://github.com/Theano/Theano/blob/0.8.X/theano/sandbox/cuda/opt.py#L1728-L1735